docs/demo/demo.md

```diff
@@ -1,233 +0,0 @@
-# Demo
-
-This section provides a detailed walkthrough of the "developer
```

Comment:
I hope this still exists someplace

Reply:
It doesn't because the text here is stale, and the demo has been ported to a packaged demo with docs.